llvm/docs/LangRef.rst

```diff
@@ -5229,6 +5229,11 @@ flag that indicates whether or not the inline asm expression has side effects,
 and a flag indicating whether the function containing the asm needs to align its
 stack conservatively.
 
+LLVM will treat the actual instructions entirely opaquely (i.e. no optimizations
+will be performed based on the contents of the template string). Only the
+operand constraints are used to deduce what the expression may do during
+execution.
```

Comment:
Second sentence should either be omitted or expanded (also relevant are inlineasm flags and attributes on the call).

Reply:
Like this?

I can also remove the sentence entirely if you prefer that.